js/news.json

```diff
@@ -1,6 +1,16 @@
 {
     "newsContent": [
-         {
+        {
+            "id": "71",
+            "newsId": "new71",
+            "newsTitle": "Accepting Nominations for the Steering Committee",
+            "newsShortTitle": "Accepting Nominations for the Steering Committee",
+            "newsDescription": "The ONNX Steering Committee is elected every year by the Contributors of ONNX project. The candidates for the Steering Committee are self-nominated, and they are not required to be Contributors. The candidate information gathered with this form will be shared publicly on 4/17/2023 (Monday). Contributors will have access to the list of candidates on ONNX Github and will be able to participate in the Steering Committee election on 5/1/2023 (Monday).",
```

Comment:
Should add that nominations are accepted until April 14

Reply:
good point. I updated that.